include/ofi_shm.h

```diff
@@ -126,6 +126,7 @@ struct smr_msg_hdr {
 	};
 } __attribute__ ((aligned(16)));
 
+#define SMR_BUF_BATCH_MAX	64
```

Comment:
The maximum number of SAR buffer is limited to 64 even if the number of peers allow more than that. We need a maximum limit because it also impacts the number of DSA commands needed. 64 gives us the best performance and going beyond does not provide meaningful benefit.